perf/bonnie.py

```diff
@@ -46,7 +47,8 @@ def setUp(self):
         """
 
         self.scratch_dir = self.params.get('dir', default=self.srcdir)
-        self.uid_to_use = self.params.get('uid-to-use', default=None)
+        self.username = getpass.getuser()
+        self.uid_to_use = self.params.get('uid-to-use', default=self.username)
```

Comment:
`self.username` is not being used anywhere outside `setUp()`. No need to create it as an instance variable. Actually, no need to create it at all:
```python
from getpass import getuser
...

        self.uid_to_use = self.params.get('uid-to-use', default=getuser())
```